### DISCUSSION TITLE
Unable to signIn - It gives error -

### DISCUSSION THREAD

**Discussion by bsuitesoftware** [vercel/community#3261] [Help] [answered]
### Summary

Please note that i am unable to login to Vercel
I verified my email successfully.
But still unable to login. 
My account - bsuitesoftware@gmail.com

### Steps to Reproduce

Login with Email , Enter bsuitesoftware@gmail.com

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. You can follow the status page for more updates https://www.vercel-status.com/incidents/590pcktngq7z
